core/src/services/s3/writer.rs

```diff
@@ -158,7 +158,16 @@ impl oio::MultipartWrite for S3Writer {
         let status = resp.status();
 
         match status {
-            StatusCode::OK => Ok(()),
+            StatusCode::OK => {
+                let (_, mut body) = resp.into_parts();
+                let bs = body.copy_to_bytes(body.remaining());
```

Comment:
`Buffer` itself implements `bytes::Buf` so we don't need to copy to bytes first.

Reply:
resolved in [3bad367](https://github.com/apache/opendal/pull/4735/commits/3bad36762f174afa3c50b186c59087532a54824e)